src/providers/seventv/SeventvEmotes.cpp

```diff
@@ -131,114 +139,44 @@ boost::optional<EmotePtr> SeventvEmotes::emote(const EmoteName &name) const
+            auto emotes = this->emotes();
+            auto pair = parseGlobalEmotes(result.parseJsonArray(), *emotes);
             if (pair.first)
                 this->global_.set(
                     std::make_shared<EmoteMap>(std::move(pair.second)));
             return pair.first;
+            return Failure;
         })
         .execute();
 }
 
 void SeventvEmotes::loadChannel(std::weak_ptr<Channel> channel,
                                 const QString &channelId,
+                                const QString &channelDisplayName,
                                 std::function<void(EmoteMap &&)> callback,
                                 bool manualRefresh)
 {
-    qCDebug(chatterinoSeventv)
-        << "Reloading 7TV Channel Emotes" << channelId << manualRefresh;
-
-    QJsonObject payload, variables;
-
-    QString query = R"(
-        query loadUserEmotes($login: String!) {
-            user(id: $login) {
-                emotes {
-                    id
-                    name
-                    provider
-                    provider_id
-                    visibility
-                    mime
-                    owner {
-                        id
-                        display_name
-                        login
-                        twitch_id
-                    }
-                }
-            }
-        })";
+    qCDebug(chatterinoSeventv) << "[7TVEmotes] Reloading 7TV Channel Emotes"
+                               << channelId << manualRefresh;
 
-    variables.insert("login", channelId);
-
-    payload.insert("query", query.replace(whitespaceRegex, " "));
-    payload.insert("variables", variables);
-
-    qDebug() << QJsonDocument(payload).toJson(QJsonDocument::Compact);
-
-    NetworkRequest(apiUrlGQL, NetworkRequestType::Post)
-        .timeout(20000)
-        .header("Content-Type", "application/json")
-        .payload(QJsonDocument(payload).toJson(QJsonDocument::Compact))
+    NetworkRequest("https://api.7tv.app/v2/users/" + channelDisplayName +
+                   "/emotes")
         .onSuccess([callback = std::move(callback), channel, channelId,
                     manualRefresh](NetworkResult result) -> Outcome {
-            QJsonObject parsedEmotes = result.parseJson()
-                                           .value("data")
-                                           .toObject()
-                                           .value("user")
-                                           .toObject();
-
-            auto emoteMap = parseChannelEmotes(parsedEmotes, channelId);
+            auto emoteMap =
+                parseChannelEmotes(result.parseJsonArray(), channelId);
             bool hasEmotes = !emoteMap.empty();
 
             qCDebug(chatterinoSeventv)
-                << "Loaded 7TV Channel Emotes" << channelId << emoteMap.size()
-                << manualRefresh;
+                << "[7TVEmotes] Loaded 7TV Channel Emotes" << channelId
+                << emoteMap.size() << manualRefresh;
```

Comment:
This is an unnecessary change - logging categories exist for a reason and are there to not hardcode categories like this.